### DISCUSSION TITLE
Use VNC to get GUI?

### DISCUSSION THREAD

**Discussion by raspiduino** [TrungNguyen1909/qemu-t8030#58] [Ideas]
I found that [Veency](https://github.com/niknah/veency) is a VNC server for iOS (require jailbreak actually), so is that possible to use this with the emulated iOS in `qemu-t8030`?

Thanks!

**Comment by iarchiveml** [comment]
It's not that we don't have a screen. We don't have a framebuffer or a GPU. This won't work until we have those.

**Reply by raspiduino** [reply]
Ok thanks!
